packs/mmonit/sensors/events_sensor.yaml

```diff
@@ -0,0 +1,22 @@
+---
+  class_name: "MmonitEventsSensor"
+  entry_point: "events_sensor.py"
+  description: "Sensor which monitors mmonit for events"
+  poll_interval: 10
+  trigger_types:
+    -
+      name: "new_event"
+      description: "Trigger which indicates that a new event has arrived at mmonit"
+      payload_schema:
+        type: "object"
+        properties:
+          host:
+            type: "string"
+          id:
+            type: "integer"
+          event:
```

Comment:
Is the set of possible values of event known upfront? If so instead of a wrapper new_event you could also choose to break it out. Would make it easier to consume in a rule.